source/mainnet/net/resources/ccd-scan.rst

```diff
@@ -20,4 +20,56 @@ CCDScan
 Delegation and baker pools
 ==========================
 
-The CCDScan tool allows bakers to view information that helps them to :ref:`manage their baker and pool responsibly<baker-pool>`. It also allows potential delegators to :ref:`research bakers and pools<baker-pool>` to find out which one is reliable and trustworthy.
+The CCDScan tool allows bakers to view information that helps them to :ref:`manage their baker and pool responsibly<baker-pool>`. It also allows potential delegators to :ref:`research bakers and pools<baker-pool>`.
+
+Use CCDScan
+===========
+
+CCDScan shows information about :ref:`blocks<blocks-view>`, :ref:`transactions<transactions-view>`, :ref:`accounts<accounts-view>`, and :ref:`staking<bakers-view>`.
+
+Home
+----
+
+On the Home page that appears when you navigate to CCDScan, you see the following information:
+
+.. image:: ../images/ccd-scan/ccd-scan-overview.png
+
+|
+
+1.  Mainnet/Testnet selector. The information shown is dependent on whether you have selected Mainnet or Testnet.
+2.  Search field: search across all relevant blockchain data for specific information, such as a particular account, transaction, block, or baker and see all relevant details associated with that entity.
+3.  Site naviation: change between pages to see other information. On a smaller screen, the page selector is available in the hamburger menu |hamburger|.
+4.  Time range filter: controls the time range of the data shown.
+5.  Total CCD in existence on either Mainnet or Testnet; on Mainnet Total CCD in circulation is also shown.
+6.  Blocks added: number of blocks added in the selected time range.
+7.  Block Time: average time since last block created in the selected time range.
+8.  Transactions: sum of transactions in the selected time range.
```

Comment:
What does sum mean? Total number of transactions in the selected time range? I suggest rephrasing to make clearer.